
Update to latest Mesa API.

  - Rene Rebe <rene@exactcode.de>

diff -ur xorg-server-0.99.1/GL/glx/g_disptab_EXT.c xorg-server-0.99.1-fixed/GL/glx/g_disptab_EXT.c
--- xorg-server-0.99.1/GL/glx/g_disptab_EXT.c	2005-08-01 22:15:45.000000000 +0200
+++ xorg-server-0.99.1-fixed/GL/glx/g_disptab_EXT.c	2005-10-07 18:29:47.000000000 +0200
@@ -2225,13 +2225,8 @@
 	__glXNoSuchRenderOpcode, /* 4219 */
 #ifndef MISSING_GL_EXTS
 	__glXDisp_ActiveStencilFaceEXT, /* 4220 */
-# ifndef __DARWIN__
-	__glXDisp_PointParameteri,      /* 4221 */
-	__glXDisp_PointParameteriv,     /* 4222 */
-# else
 	__glXDisp_PointParameteriNV,    /* 4221 */
 	__glXDisp_PointParameterivNV,   /* 4222 */
-# endif
 #else
 	__glXNoSuchRenderOpcode,
 	__glXNoSuchRenderOpcode,
@@ -4435,13 +4430,8 @@
 	__glXNoSuchRenderOpcode, /* 4219 */
 #ifndef MISSING_GL_EXTS
 	__glXDispSwap_ActiveStencilFaceEXT, /* 4220 */
-# ifndef __DARWIN__
-	__glXDispSwap_PointParameteri,      /* 4221 */
-	__glXDispSwap_PointParameteriv,     /* 4222 */
-# else
 	__glXDispSwap_PointParameteriNV,    /* 4221 */
 	__glXDispSwap_PointParameterivNV,   /* 4222 */
-# endif
 #else
 	__glXNoSuchRenderOpcode,
 	__glXNoSuchRenderOpcode,
diff -ur xorg-server-0.99.1/GL/glx/g_disptab_EXT.h xorg-server-0.99.1-fixed/GL/glx/g_disptab_EXT.h
--- xorg-server-0.99.1/GL/glx/g_disptab_EXT.h	2005-08-01 22:15:45.000000000 +0200
+++ xorg-server-0.99.1-fixed/GL/glx/g_disptab_EXT.h	2005-10-07 18:29:01.000000000 +0200
@@ -99,17 +99,10 @@
 extern void __glXDisp_BlendFuncSeparate(GLbyte *);
 extern void __glXDispSwap_BlendFuncSeparate(GLbyte *);
 
-#ifdef __DARWIN__
 extern void __glXDisp_PointParameteriNV(GLbyte *);
 extern void __glXDisp_PointParameterivNV(GLbyte *);
 extern void __glXDispSwap_PointParameteriNV(GLbyte *);
 extern void __glXDispSwap_PointParameterivNV(GLbyte *);
-#else
-extern void __glXDisp_PointParameteri(GLbyte *);
-extern void __glXDisp_PointParameteriv(GLbyte *);
-extern void __glXDispSwap_PointParameteri(GLbyte *);
-extern void __glXDispSwap_PointParameteriv(GLbyte *);
-#endif
 
 extern void __glXDisp_ActiveStencilFaceEXT(GLbyte*);
 
diff -ur xorg-server-0.99.1/GL/glx/g_render.c xorg-server-0.99.1-fixed/GL/glx/g_render.c
--- xorg-server-0.99.1/GL/glx/g_render.c	2005-08-01 22:15:45.000000000 +0200
+++ xorg-server-0.99.1-fixed/GL/glx/g_render.c	2005-10-07 17:43:17.000000000 +0200
@@ -2254,12 +2254,12 @@
 }
 
 #ifdef __DARWIN__
-#define __glPointParameterivNV_size __glPointParameteriv_size
+#define __glPointParameterivNV_size __glPointParameterivNV_size
 dispatch_template_enum1_1s(PointParameteriNV,  GLint)
 dispatch_template_enum1_Vv(PointParameterivNV, GLint)
 #else
-dispatch_template_enum1_1s(PointParameteri,  GLint)
-dispatch_template_enum1_Vv(PointParameteriv, GLint)
+dispatch_template_enum1_1s(PointParameteriNV,  GLint)
+dispatch_template_enum1_Vv(PointParameterivNV, GLint)
 #endif
 
 void __glXDisp_ActiveStencilFaceEXT(GLbyte *pc)
diff -ur xorg-server-0.99.1/GL/glx/g_renderswap.c xorg-server-0.99.1-fixed/GL/glx/g_renderswap.c
--- xorg-server-0.99.1/GL/glx/g_renderswap.c	2005-08-01 22:15:45.000000000 +0200
+++ xorg-server-0.99.1-fixed/GL/glx/g_renderswap.c	2005-10-07 17:39:48.000000000 +0200
@@ -3339,7 +3339,7 @@
 
 	__GLX_SWAP_INT(pc + 0);
 	pname = *(GLenum *)(pc + 0);
-	compsize = __glPointParameterfvARB_size(pname);
+	compsize = __glPointParameterfvEXT_size(pname);
 	if (compsize < 0) compsize = 0;
 	__GLX_SWAP_FLOAT_ARRAY(pc + 4, compsize);
 
diff -ur xorg-server-0.99.1/GL/glx/impsize.h xorg-server-0.99.1-fixed/GL/glx/impsize.h
--- xorg-server-0.99.1/GL/glx/impsize.h	2005-08-01 22:15:45.000000000 +0200
+++ xorg-server-0.99.1-fixed/GL/glx/impsize.h	2005-10-07 17:39:48.000000000 +0200
@@ -71,7 +71,7 @@
 extern int __glTexParameteriv_size(GLenum e);
 extern int __glEvalComputeK(GLenum target);
 
-extern int __glPointParameterfvARB_size(GLenum e);
-extern int __glPointParameteriv_size(GLenum e);
+extern int __glPointParameterfvEXT_size(GLenum e);
+extern int __glPointParameterivNV_size(GLenum e);
 
 #endif /* _impsize_h_ */
diff -ur xorg-server-0.99.1/GL/glx/rensize.c xorg-server-0.99.1-fixed/GL/glx/rensize.c
--- xorg-server-0.99.1/GL/glx/rensize.c	2005-08-01 22:15:45.000000000 +0200
+++ xorg-server-0.99.1-fixed/GL/glx/rensize.c	2005-10-07 17:39:48.000000000 +0200
@@ -922,7 +922,7 @@
     if (swap) {
 	pname = SWAPL( pname );
     }
-    return 4 * __glPointParameterfvARB_size( pname );
+    return 4 * __glPointParameterfvEXT_size( pname );
 }
 
 int __glXPointParameterivReqSize(GLbyte *pc, Bool swap )
